pg_diffix/query/allowed_functions.h

```diff
@@ -28,4 +30,10 @@ extern bool is_substring_builtin(Oid funcoid);
  */
 extern bool is_implicit_range_builtin_untrusted(Oid funcoid);
 
+/*
+ * Returns whether the OID points to a relation in the `pg_catalog`, for which accessing the
+ * `selected_cols` is allowed.
```

Comment:
Will the formatter revert back if changed like this?

```suggestion
 * Returns whether the OID points to a relation in the `pg_catalog`,
 * for which accessing the `selected_cols` is allowed.
```

Reply:
I'm not sure I understand the function comment in either case.

It should just say whether the given catalog table and select columns are allowed.

Also the `rte` in the name is redundant.

Reply:
Not the formatter but the rewrapper will make it a single line. For the sake of moving forward we can take your version.

Reply:
re second comment, you're right, something went wrong there

Reply:
ok, how about now?